unic/ucd/name/src/name.rs

```diff
@@ -47,19 +70,21 @@ impl Name {
 
     /// Length of the name in bytes.
     pub fn len(&self) -> usize {
-        match self {
-            &Name::NR1(ch) => {
+        match *self {
+            Name::NR1(ch) => {
                 let mut len = PREFIX_HANGUL_SYLLABLE.len();
-                // FIXME: use decomposed jamo short names instead
-                len += Name::number_of_hex_digits(ch);
+                {
+                    let mut count_jamos = |_| len += 1;
```

Comment:
Some of the jamo short names in the table are more than one character long -- is this correct?

(I really have no idea how hangul is handled in Unicode...)

Reply:
Nice catch!
Yeah, some of them are more than one and some of them are less than one (empty).
I will change this to use the length of the jamo.